transport/sqs/src/main/kotlin/SqsMessageReceiverFactory.kt

```diff
@@ -52,7 +52,7 @@ class SqsMessageReceiverFactory : MessageReceiverFactory {
         from: Endpoint<T>,
         configManager: ConfigManager,
         handler: EndpointHandler<T>
-    ) = runBlocking {
+    ) = coroutineScope {
```

Comment:
Couldn't this simply be omitted now, and below change `isActive` to `coroutineContext.isActive`?

Reply:
Yes, I assume this is equivalent. Changed accordingly.